### PR TITLE
add docstrings

### PR DESCRIPTION
**Pull request by rpitonak**

### CODE REVIEW

**Comment by marusinm** [comment]
LGTM

File: sentinel_downloader/api/api.py [hunk 1]
```diff
@@ -54,6 +75,13 @@ def download(self):
             self.download_image(time, path)
 
     def download_image(self, time, path=None):
+        """
+        Download images from single time range.
```
Comment (mjavorka) [inline sentinel_downloader/api/api.py]: Shouldn't be an empty line between description and `:params`?

File: sentinel_downloader/api/api.py [hunk 2]
```diff
@@ -54,6 +75,13 @@ def download(self):
...
+        :param time: time range in format ('time_from', 'time_to')
+        :param path: path where to save images
+
```
Comment (mjavorka) [inline sentinel_downloader/api/api.py]: Also, no empty line between `:param` and `:return`?

File: sentinel_downloader/config.py [hunk 3]
```diff
@@ -46,6 +50,12 @@ def __init__(self):
 
     @staticmethod
     def validate(raw_dict: dict) -> None:
+        """
+        validate config file according to schema.py
```
Comment (mjavorka) [inline sentinel_downloader/config.py]: The description should start with capital letter.

File: sentinel_downloader/config.py [hunk 4]
```diff
@@ -54,6 +64,12 @@ def validate(raw_dict: dict) -> None:
 
     @staticmethod
     def get_from_dict(raw_dict: dict, validate=True) -> "Config":
+        """
+        Get Config object from python dict
```
Comment (mjavorka) [inline sentinel_downloader/config.py]: Empty line and a dot at the end of the sentence.

File: sentinel_downloader/config.py [hunk 5]
```diff
@@ -75,6 +91,11 @@ def get_from_dict(raw_dict: dict, validate=True) -> "Config":
 
     @staticmethod
     def get_config(path=None) -> "Config":
+        """
+        Load configuration file from provided path or current working directory.
```
Comment (mjavorka) [inline sentinel_downloader/config.py]: Empty line

**Pull request merged**